mypy/subtypes.py

```diff
@@ -146,10 +146,9 @@ def visit_instance(self, left: Instance) -> bool:
             rname = right.type.fullname()
             if not left.type.has_base(rname) and rname != 'builtins.object':
                 return False
-
             # Map left type to corresponding right instances.
             t = map_instance_to_supertype(left, right.type)
-
+            # TODO: assert len(t.args) == len(right.args)
```

Comment:
> Or maybe I'll just sneak in some no-fixture tests into the test suite while nobody's watching.

:-) I like the idea of running test with real full stubs, but let us postpone this for some time.
It is better to update fixtures for now, to make them match real stubs more.

Actually, the fact that some stubs _can_ crash here is a bit dangerous, maybe we are still missing something?